src/libraries/System.Text.Json/tests/Common/ConstructorTests/ConstructorTests.Stream.cs

```diff
@@ -155,7 +156,7 @@ async Task RunTestAsync<T>()
                         DefaultBufferSize = 1
                     };
 
-                    await Assert.ThrowsAsync<NotSupportedException>(async () => await StreamingSerializer.DeserializeWrapper<T>(stream, options));
+                    value = await StreamingSerializer.DeserializeWrapper<T>(stream, options);
```

Comment:
should you add at least some tiny sanitization for couple of values that they look good? (same above)

Reply:
The problem is that the test method is generic on the deserialized type, so you can't really check anything other than validate that the JSON is nontrivial. I'm validating that roundtripping works as expected in the new test.